config/menus/options.cfg

```diff
@@ -1,6 +1,9 @@
+if (getalias selectedaspectratio) [] [ selectedaspectratio = 1 ]
+setpersist selectedaspectratio 1
+setcomplete selectedapsectratio 1
```

Comment:
Why do you `setcomplete`? Should people be accessing this alias from the console?

Reply:
tbh, I don't even knew what it did, but I did it because the ones beneigh it did too

Reply:
`setcomplete` is used to allow various forms of tab-completion; this one makes the alias itself tab-completable

Reply:
oooh, ok that makes sense, thx